### ISSUE TITLE
Resolver is Deprecated

### ISSUE THREAD

**Issue by hmlongco** [hmlongco/Resolver#177]
Resolver is now officially deprecated and replaced by my new dependency injection system, [Factory](https://github.com/hmlongco/Factory).

Factory is a true container-based dependency injection system that's compile-time safe and is smaller, lighter, and faster than Resolver. 

As good as Resolver was, **Factory is better.**

**Comment by cyonix** [comment]
@hmlongco I agree that Factory is indeed better. Are there any reasons why you haven't archived Resolver yet?

**Comment by hmlongco** [comment]
Some people still use it.
